kpops/components/streams_bootstrap/producer/producer_app.py

```diff
@@ -68,6 +68,14 @@ def apply_to_outputs(self, name: str, topic: TopicConfig) -> None:
             case _:
                 super().apply_to_outputs(name, topic)
 
+    @property
```

Comment:
please mark as override

Reply:
I needed to remove it because pyright doesn't want to have it.

Reply:
did you set it in the correct order?
```python
@property
@override
def ...
```

Reply:
this should be all. I hope!

Reply:
well it seems that regardless of the order, pyright doesn't like it

Reply:
should I remove it ?

Reply:
these are not in the correct order https://github.com/bakdata/kpops/pull/312/commits/58cc85ffc4c4837479f72698bfcaab051d2d9a4f

Reply:
https://github.com/bakdata/kpops/actions/runs/7623207572/job/20762773438#step:8:20 the order is still not the problem https://github.com/bakdata/kpops/blob/feature/parallelization/kpops/components/base_components/pipeline_component.py#L75

Reply:
ah, check the import, there is a mistake. right now it only gets imported in the except clause